File: problems/euler-2-even-fibonacci-numbers.py/euler-2-even-fibonacci-numbers.py
#!/usr/bin/env python3

import sys

n = int(sys.argv[1])

total = 0
prev = 1
curr = 1

while curr < n:
    if curr % 2 == 0:
        total += curr

    tmp = curr
    curr += prev
    prev = tmp

print(total)

File: problems/euler-2-even-fibonacci-numbers.js/euler-2-even-fibonacci-numbers.js
const n = parseInt(process.argv[2]);

let total = 0;
let prev = 1;
let curr = 1;

while (curr < n) {
    if (curr % 2 === 0) {
        total += curr;
    }

    const tmp = curr;
    curr += prev;
    prev = tmp;
}

console.log(total);
